honeybee_radiance/primitive/material/glow.py

```diff
@@ -0,0 +1,231 @@
+"""Radiance Glow Material.
+
+http://radsite.lbl.gov/radiance/refer/ray.html#Glow
+"""
+
+from .materialbase import Material
+import honeybee_radiance.typing as typing
+
+
+class Glow(Material):
+    """Create glow material."""
+
+    def __init__(self, name, red=0.0, green=0.0, blue=0.0, max_radius=0.0,
+                 modifier='void', dependencies=None):
+        """Init Glow material.
+
+        args:
+            name: Material name as a string. The name should not have whitespaces or
+                special characters.
+            red: A positive value for the Red channel of the glow (default: 0).
```

Comment:
This one I don't know. Based on what I can see there is no limitation and we had it as a positive number. Let me check the source code and see if I can find a reference to say which one is correct.

Reply:
I found this in documentation which indicates it can be larger than 1. I can't say that I fully understand why is this but this confirms that the docstring here is fine.

https://github.com/NREL/Radiance/blob/80dc201330e60535170151ed99bfeec1cbc2bcf8/test/renders/sunset_sky.rad#L10-L13


@rpg777, @sariths any comments on this? It looks like radiance has no limitations / value checking for input values. I first tried to figure this out by trying to crash radiance but it never failed. I even tried negative values for glass transmissivity and it worked!

Reply:
That's right, there is no error checking of inputs in Radiance. You can do negative transmissivity, you can do reflectance > 1, effectively making magical surfaces that create energy. Fun!